### DISCUSSION TITLE
Any plans for a generic search function?

### DISCUSSION THREAD

**Discussion by rodvand** [netbox-community/netbox#6362] [Ideas]
I have noticed the search options on each object overview is very subjective, and only implemented after a feature request is filed. 

Could it be an idea to make this more generic, and have the user be able to add/remove filters (add to user preferences)? That way a number of issues are prevented and users can configure their own search "side bar".

**Comment by jeremystretch** [comment]
We're aware of the need for a "real" search solution, but that's about as far as the conversation has gotten.
